docs/Using-Docker.md

```diff
@@ -45,13 +45,15 @@ Replace `<image-name>` with a name for the Docker image, e.g. `balance.ball.v0.1
 
 Run the Docker container by calling the following command at the top-level of the repository:
 ```
-docker run --mount type=bind,source="$(pwd)"/unity-volume,target=/unity-volume \
+docker run --name <container-name> \
+	 --mount type=bind,source="$(pwd)"/unity-volume,target=/unity-volume \
      <image-name>:latest <environment-name> \
      --docker-target-name=unity-volume \
      --train --run-id=<run-id>
 ```
 
 Notes on argument values:
+- `<container-name>` is used to identify the container (in case you want to terminate it later or examine its logs). This is optional and Docker will set a random string if this is not set. _Note that this must be unique for every run of a Docker Image_
```

Comment:
`or examine its logs` How do I examine the logs ? Is it a Docker thing ? If yes, maybe it should not be here ?

Reply:
It is a docker thing. Yes, agreed, it can be eliminated for now.